lib/mayaHydra/hydraExtensions/sceneIndex/mayaHydraSceneIndex.cpp

```diff
@@ -183,6 +183,16 @@ namespace {
         return mayaPath;
     }
 
+    bool IsTexturedSkyDomeRenderItem(const SdfPath& name)
+    {
+        static const std::string _aiTexturedSkyDome("texturedSkyDome");
+        if (name.GetString().find(_aiTexturedSkyDome) != std::string::npos) {
```

Comment:
I am not sure this is enough as they could be named another way that texturedSkyDome, right ?
The name probably comes from the maya node which can be named anything if I am correct ?

Reply:
To my understanding, this is a OGS generated render item with a hardcoded name. OGS generated RI with certain naming conventions:
For textured dome mesh render item:
/MayaHydraViewportRenderer/rprims/aiSkyDomeLight1/aiSkyDomeLightShape1/texturedSkyDome_941
For wireframe render item:
/MayaHydraViewportRenderer/rprims/aiSkyDomeLight1/aiSkyDomeLightShape1/active_wireframeSkyDome_9417

I tested renaming the domelight and suffix "../../texturedSkyDome_xxx" remains.